src/main/java/com/codeofus/rent_a_park/errors/BadRequestException.java

```diff
@@ -0,0 +1,23 @@
+package com.codeofus.rent_a_park.errors;
+
+public class BadRequestException extends Throwable {
```

Comment:
Why extend Throwable?

Reply:
because of `throw new BadRequestException` in one method